content/lessons/chapter-3/pool-1.tsx

```diff
@@ -59,7 +59,7 @@ export default function Pool1({ lang }) {
   const ANTAGONISTS = [
     {
       username: 'BitRey',
-      avatar: '/assets/avatars/5.png',
+      avatar: '/assets/avatars/bitrey.png',
```

Comment:
I think for 'You' we should use the avatar of the currently logged in user (if it exists) and fall back to 1.png or perhaps a default profile icon or something:

![image](https://github.com/saving-satoshi/saving-satoshi/assets/10757768/309f4da5-1207-48aa-a392-88ecfc930fb2)